l10n/en-US/viewer.ftl

```diff
@@ -323,8 +323,14 @@ pdfjs-editor-ink-button-label = Draw
 pdfjs-editor-stamp-button =
     .title = Add or edit images
 pdfjs-editor-stamp-button-label = Add or edit images
-pdfjs-editor-remove-button =
-    .title = Remove
+
+# Remove button for the various kind of editor.
```

Comment:
Sorry, missed the comment here.

This is a single string comment that only applies to the next string. You want a group-level comment (two hashes), if you want it to show up in Pontoon, or a stand-alone comment (one hash, leave an empty line after), which won't be displayed to localizers. If you go with the group-level comment, you also need to "reset it" (add an empty group-level comment after the strings).

Reply:
With a group comment, it would look like this (notice the empty lines before and after).

```

## Remove button for the various kinds of editor.

pdfjs-editor-remove-ink-button =
    .title = Remove drawing
pdfjs-editor-remove-freetext-button =
    .title = Remove text
pdfjs-editor-remove-stamp-button =
    .title = Remove image

##

# Editor Parameters
pdfjs-editor-free-text-color-input = Color
```